### DISCUSSION TITLE
Enhancement:

### DISCUSSION THREAD

**Discussion by JorosTheModderPH** [FunkinCrew/Funkin#1661] [General]
Released the source code of week 7 please :D

**Comment by ghost** [comment]
They're working on cutscenes for desktop
have patience

**Comment by ghost** [comment]
its somewhere on github

**Comment by ghost** [comment]
That is the **compiled HTML5 version**. The source does not exist yet.

**Comment by ghost** [comment]
> That is the **compiled HTML5 version**. The source does not exist yet.

i mean it does contain the assets

**Comment by ghost** [comment]
Yeah true but that's not the source. The compiled version must contain assets in their own folder due to how HaxeFlixel works, and cutscenes do not work on desktop (can't be compiled)

**Comment by suprstarrd** [comment]
@nickpythonluajava mine, and many others, as has already been stated, is just the compiled HTML5 version. no source available

**Comment by bonkmaykr** [comment]
It's not finished. Just be patient.

They want to keep the release a surprise.

**Comment by GOOBOZA** [comment]
> They're working on cutscenes for desktop
> have patience

thats not why it is taking a long they're making it with week 8
